corehq/apps/domain/migrations/0008_geocoder.py

```diff
@@ -0,0 +1,26 @@
+from django.db import migrations
+from django.core.management import call_command
+from corehq.util.django_migrations import skip_on_fresh_install
+
+from corehq.privileges import GEOCODER
+
+
+@skip_on_fresh_install
+def _grandfather_basic_privs(apps, schema_editor):
+    call_command('cchq_prbac_bootstrap')
+    call_command(
+        'cchq_prbac_grandfather_privs',
+        GEOCODER,
+        skip_edition='Paused,Community,Standard,Pro,Advanced',
```

Comment:
I don't think this should skip advanced plans. I don't know the accounting code very well, that comment is based on looking at https://github.com/dimagi/commcare-hq/commit/b22aba39401af264282acc8f140894ce8cd1206f